peer/store-utils-mock.go

```diff
@@ -0,0 +1,19 @@
+package peer
+
+// This file contains mock functions to be used during unit tests.
```

Comment:
The mock functions provided here are for functions that are exported by this package. They are for other use in unit tests of other packages that make use of this package and functions exported by it.
Instead of dumping these mock functions into a separate test package containing all other mocked functions, it stay in this package itself close to the functions being mocked.